src/vs/workbench/services/extensions/browser/webWorkerExtensionHost.ts

```diff
@@ -109,12 +110,19 @@ export class WebWorkerExtensionHost extends Disposable implements IExtensionHost
 				const res = new URL(`${baseUrl}/out/${iframeModulePath}${suffix}`);
 				res.searchParams.set('parentOrigin', mainWindow.origin);
 				res.searchParams.set('salt', stableOriginUUID);
+				if (locale !== 'en') {
```

Comment:
Sorry, also in platform there is a LANGUAGE_DEFAULT which you can use instead of hardcoding 'en'